app/Http/Controllers/SettingsController.php

```diff
@@ -203,8 +204,10 @@ public function exportToSql()
 
         $adapter = disk_adapter(ExportAccountAsSQL::STORAGE);
 
+        $exportdate = date('Y-m-d');
```

Comment:
It will be better like this:
```suggestion
        $exportdate = Carbon::now(DateHelper::getTimezone())->format('Y-m-d');
```